Core/Core/Conversations/APIConversation.swift

```diff
@@ -248,3 +248,23 @@ public struct PostAddMessageRequest: APIRequestable {
     public var path: String { "conversations/\(conversationID)/add_message" }
     public let method = APIMethod.post
 }
+
+public struct PostConversationRequest: APIRequestable {
+    public typealias Response = [APIConversation]
+
+    public struct Body: Encodable {
+        public let subject: String
+        public let body: String
+        public let recipients: [String]
+        public let context_code: String?
+        public let media_comment_id: String? = nil
+        public let media_comment_type: MediaCommentType? = nil
+        public let attachment_ids: [String]? = nil
+        public let group_conversation: Bool? = true
```

Comment:
@armstrongnate the request is already defaulting `group_conversation = true`

Reply:
Correct. I will take a look at that.

Reply:
These Jan 15 conversations were all from the same message.

![IMG_B13829831E3B-1](https://user-images.githubusercontent.com/1187499/72456665-5fc14800-3782-11ea-88c8-acdafafc2c2c.jpeg)